index.js

```diff
@@ -0,0 +1,231 @@
+import BookList from './modules/bookList.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
+const bList = new BookList();
+const bookTitle = document.getElementById('bookTilte');
+const bookAuthor = document.getElementById('bookAuthor');
+const btnAdd = document.getElementById('Add');
+const showList = document.getElementById('showList');
+const showBook = document.getElementById('bookList');
+const addBook = document.getElementById('addBook');
+const contact = document.getElementById('contact');
+const showAdd = document.getElementById('showAdd');
+const showCon = document.getElementById('showCon');
+const time = document.getElementById('time');
+
+const showBooks = () => {
+  showBook.classList.remove('Invisible');
+  showBook.classList.add('visible');
+  showList.classList.add('colBlue');
+
+  if (addBook.classList.contains('visible')) {
+    addBook.classList.remove('visible');
+    addBook.classList.add('Invisible');
+    showAdd.classList.remove('colBlue');
+  }
+  if (contact.classList.contains('visible')) {
+    contact.classList.remove('visible');
+    contact.classList.add('Invisible');
+    showAdd.classList.remove('colBlue');
+  }
+};
+
+const AddShow = () => {
+  addBook.classList.remove('Invisible');
+  addBook.classList.add('visible');
+  showAdd.classList.add('colBlue');
+
+  if (showBook.classList.contains('visible')) {
+    showBook.classList.remove('visible');
+    showBook.classList.add('Invisible');
+    showList.classList.remove('colBlue');
+  }
+  if (contact.classList.contains('visible')) {
+    contact.classList.remove('visible');
+    contact.classList.add('Invisible');
+    showCon.classList.remove('colBlue');
+  }
+};
+
+const showContact = () => {
+  contact.classList.remove('Invisible');
+  contact.classList.add('visible');
+  showCon.classList.add('colBlue');
+
+  if (showBook.classList.contains('visible')) {
+    showBook.classList.remove('visible');
+    showBook.classList.add('Invisible');
+    showList.classList.remove('colBlue');
+  }
+  if (addBook.classList.contains('visible')) {
+    addBook.classList.remove('visible');
+    addBook.classList.add('Invisible');
+    showAdd.classList.remove('colBlue');
+  }
+};
+
+const Add = () => {
+  if (bookTitle.value !== '' && bookAuthor.value !== '') {
+    bList.AddBook(bookTitle.value, bookAuthor.value);
+    localStorage.setItem('BookList', JSON.stringify(bList.Books));
+    window.location.reload();
+  }
+};
+
+const Remove = () => {
+  if (this.id > -1) {
+    bList.RemoveBook(this.id);
+    localStorage.setItem('BookList', JSON.stringify(bList.Books));
+    window.location.reload();
+  }
+};
```

Comment:
- [ ] It's not possible to add and remove a book. It through undefined.

![Screen Shot 2022-02-21 at 16 15 13](https://user-images.githubusercontent.com/67211919/155032507-4479f5b2-b470-404b-b15c-4c964e761bef.png)

Reply:
This issue is well resolved! ✔️